nf_core/utils.py

```diff
@@ -1136,6 +1136,58 @@ def load_tools_config(directory: Union[str, Path] = ".") -> Tuple[Optional[Path]
             error_message += f"\n{error['loc'][0]}: {error['msg']}"
         raise AssertionError(error_message)
 
+    wf_config = fetch_wf_config(Path(directory))
+    if nf_core_yaml_config["repository_type"] == "pipeline" and wf_config != {}:
+        # Retrieve information if template from config file is empty
+        config_template_keys = (
+            tools_config["template"].keys()
+            if "template" in tools_config and tools_config["template"] is not None
+            else []
+        )
```

Comment:
```suggestion
       config_template_keys = tools_config.get('template', {}).keys()
```

Reply:
the value of `template` can be None, so this syntax fails https://github.com/nf-core/tools/actions/runs/11141987288/job/30964022002?pr=3185 is there a way to take this into account? or should I revert the chagne?